src/components/common/CreateUserWizard/CreateUserWizard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { Props as FormattedMessageProps } from 'react-intl/src/components/message';
+
+import Heading from '~shared/Heading';
+import ColonyButton from '~shared/Button';
+import WizardTemplate from '~frame/WizardTemplateColony';
+import { withWizard } from '~shared/Wizard';
+
+import StepUserName from './StepUserName';
+import StepConfirmTransaction from './StepConfirmTransaction';
+import StepUserEmail from './StepUserEmail';
+
+import styles from './CreateUserWizard.css';
+
+export interface FormValues {
+  [k: string]: string | string[];
+  username: string;
+  email: string;
+  emailPermissions: string[];
+}
+
+const steps = [StepUserEmail, StepUserName, StepConfirmTransaction];
+const initialValues = [{ email: '', emailPermissions: [] }, { username: '' }];
+
+const CreateUserContainer = withWizard<FormValues>({
+  initialValues,
+  steps,
+})(WizardTemplate, { hideQR: true });
+
+interface UserStepTemplateProps {
+  heading: MessageDescriptor | string;
+  description: MessageDescriptor;
+  descriptionValues?: FormattedMessageProps['values'];
+  Input: JSX.Element;
+  Button: JSX.Element;
```

Comment:
I was trying to find examples of similar props and I think the general pattern is to use camelCase despite them accepting a JSX.Element. Please have a look at React Router's `Route` and its `element` prop, or, in the dapp, there's `ClaimFunds` and `VestingPageLayout` components that use camelCase for JSX.Element type props.

Reply:
Yep happy with that